app/pages/1_Scenario_comparison.py

```diff
@@ -37,7 +37,8 @@ def main():
     tech_map = dict(map(reversed, carriers_map.items()))
 
     ###### first dropdown plotting n.statistics #####
-    params = list(network_map.values())[0].statistics().columns
+    remove_params = set(["Capacity Factor", "Supply", "Withdrawal"])
+    params = list(set(list(network_map.values())[0].statistics().columns)-remove_params)
```

Comment:
Agree with `Supply` and `Withdrawal`, but would rather sacrifice with something like `Revenue` than with `Capacity Factor`. I feel the latter one is quite relevant for a high-level overview.